include/wil/cppwinrt_notifiable_module_lock.h

```diff
@@ -0,0 +1,103 @@
+//*********************************************************
+//
+//    Copyright (c) Microsoft. All rights reserved.
+//    This code is licensed under the MIT License.
+//    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF
+//    ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+//    TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+//    PARTICULAR PURPOSE AND NONINFRINGEMENT.
+//
+//*********************************************************
+//! @file
+//! Utilities for implementing OOP COM server with cppwinrt only
+
+#ifndef __WIL_CPPWINRT_NOTIFIABLE_MODULE_LOCK_DEFINED
+#define __WIL_CPPWINRT_NOTIFIABLE_MODULE_LOCK_DEFINED
+
+#if _MSVC_LANG >= 201703L

```

Comment:
Seems unnecessary. Isn't C++/WinRT's minimum required standard _at least_ C++17 (if not higher now)?

Reply:
Same comment on *com_server file

Reply:
Yes

Reply:
This is to guard against running test on non-C++/17 mode. I saw other tests including this macro 🧐

Reply:
This header file (and subsequent test file(s)) are standalone and intended strictly for C++/WinRT. It's therefore reasonable to assume necessary levels of compiler support. If, say, you wanted to include this header from something like `resource.h`, then you would need guards, however even then it's probably more appropriate to guard the inclusion.